src/add-remove.js

```diff
@@ -93,42 +93,48 @@ function AddTask(description) {
   taskArray.push(new Task(description));
   localStorage.setItem('taskArray', JSON.stringify(taskArray));
   addInput.value = '';
-  DisplayTask();
+  DisplayTask(taskArray);
+}
+
+function EditTesting(indexToUpdate, taskEdit) {
```

Comment:
## [Optional]
Let's try to be more professional about our naming, `EditTesting` seems confusing.

`EditedTask` could be better.